services/QuillLMS/spec/models/cron_spec.rb

```diff
@@ -44,6 +44,13 @@
       Cron.interval_1_hour
     end
 
+    it 'enqueues StudentLearningSequences::FillMissingWorker' do
```

Comment:
Slightly off-topic, but this whole file is essentially testing an external library's API.

Reply:
It's not even really doing that because `expect` means that the calls are never made.  What this file actually does, I think, is backstop us from accidentally deleting a registered job.  You're right that it's not actually testing anything.  It's a lot more like an `assert` call that will error if the `perform_async` calls aren't being made for some reason.